flow-typed/prosemirror-keymap.js

```diff
@@ -0,0 +1,5 @@
+// @flow
+
+declare module 'prosemirror-keymap' {
+  declare module.exports: any;
```

Comment:
Declaring everything as "any" defeats the whole reason static typing systems like Flow, Typescript exist.  Learn how the build system works, and import the correct type definitions